src/i18n/Streami18n.ts

```diff
@@ -532,10 +537,10 @@ export class Streami18n {
         [defaultNS]:
           this.translations[this.currentLanguage] &&
           this.translations[this.currentLanguage][defaultNS]
-            ? {
-                ...this.translations[this.currentLanguage][defaultNS],
-                ...translationsForLanguage,
-              }
+            ? defaultsDeep(
+                translationsForLanguage,
+                this.translations[this.currentLanguage][defaultNS],
+              )
```

Comment:
Translations JSON is now nested, so we need to merge in the defaults recursively.

Reply:
No longer relevant.